specification/cognitiveservices/data-plane/ContentModerator/stable/v1.0/examples/AddTermResource.JSON

```diff
@@ -7,9 +7,6 @@
     "language": "eng"
   },
   "responses": {
-    "201": {
-      "headers": {},
-      "body": {}
-    }
+    "201": {}
```

Comment:
Is there any reason that `headers` and `body` are set empty explicitly?  @davidlicig @felixwa @yangyua

Reply:
I dont think there is any explicit reason...